packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -61,6 +65,23 @@ class Send extends Component {
     }
   });
 
+  changeAmountFontSize = () => {
+    const amountElement = document.getElementById('amount');
```

Comment:
No! React uses its own virtual DOM, so we never directly modify the DOM.

Reply:
I've been looking into this but had difficulties using refs. I've also tried controlled components, and mutators unsuccessfully so far

Reply:
An idea: on `<Field name="amount">`, in its className field, you can add a CSS class when the `values.amount`'s length is longer than some value.